frontend/src/features/mapSideBar/SidebarContents/AssociatedLandForm.tsx

```diff
@@ -532,12 +532,16 @@ const AssociatedLandForm: React.FC<IAssociatedLandParentForm> = (
           onSubmit={async (values, actions) => {
             const apiValues = valuesToApiFormat(_.cloneDeep(values), keycloak.agencyId);
             try {
+              let land;
               if (!values.data.id) {
-                await createBuilding(apiValues)(dispatch);
+                land = await createBuilding(apiValues)(dispatch);
               } else {
-                await updateBuilding(apiValues)(dispatch);
+                land = await updateBuilding(apiValues)(dispatch);
+              }
+              actions.resetForm({ values: { ...values, data: land } });
+              if (apiValues.parcels.length > 0) {
+                props.setAssociatedLandComplete(true);
```

Comment:
If the user removed all of the associated land, show them the associated land starting screen so they can re-add some.

Reply:
I'm confused, isn't this checking if there is still associated land?

Reply:
setting associatedLandComplete to true displays a modal that will close or reset the sidebar. However, if there are no parcels we don't want to display this modal, but instead allow the user to continue interacting with the form after the save action.